app/lib/finder_api.rb

```diff
@@ -70,6 +76,7 @@ def fetch_search_response(content_item)
     queries = query_builder_class.new(
       finder_content_item: content_item,
       params: filter_params,
+      override_sort_for_feed: override_sort_for_feed,
```

Comment:
If you've already over-written the order argument above with 'most-recent', does `OrderQueryBuilder` also need to know that it is fetching data for an atom feed?

Reply:
Yes, as the `order` argument is only used when merging batch search results